scripts/test_pg_upgrade.py

```diff
@@ -95,11 +95,9 @@ def initialize_node(working_dir, prefix, port, bin_dir, base_dir):
 node_old.safe_psql(dbname=pg_database_test, filename="test/sql/updates/setup.check.sql")
 
 # Run new psql over the old node to have the same psql output
-node_new.port = pg_port_old
 (code, old_out, old_err) = node_new.psql(
```

Comment:
`Run new psql over the old node to have the same psql output`. Does this line still do what this comment say?

Reply:
Nope, now with `port` being read-only it will not work as is.